File: 26.10.2023/pi.py
def calculate_pi() -> float:
    """
    1/1 + 1/4 + 1/9 + 1/16 + 1/25 + ... = (pi^2)/6
    """
    a = sum(1 / (i ** 2) for i in range(1, 10_000_000))
    print(pow(a * 6, .5))

calculate_pi()

File: 26.10.2023/pi.c
#include <stdio.h>
#include <math.h>

float calculate_pi() {
    /*
    1/1 + 1/4 + 1/9 + 1/16 + 1/25 + ... = (pi^2)/6
    */
    float a = 0;
    for (int i = 1; i <= 100000000; i++) {
        a += 1 / pow(i, 2);
    }
    printf("%f\n", sqrt(a * 6));
}

int main() {
    calculate_pi();
    return 0;
}
